src/patternfly/layouts/Page/page.scss

```diff
@@ -169,14 +134,17 @@
 
 // Link
 .pf-l-page__header-brand-link {
-  display: none;
+  display: flex;
   flex: 1;
   align-items: center;
-  align-self: flex-end;
-  min-height: var(--pf-l-page__header-brand-link--MinHeight); // Hardcode link min-height so it matches the button height
 
-  @media (min-width: $pf-global--breakpoint--sm) {
-    display: flex;
+  .pf-c-brand {
+    max-height: var(--pf-l-page__header-brand-link-brand--MaxHeight); // Hardcode brand image max-height so it always aligns with everthing else.
```

Comment:
```suggestion
    max-height: var(--pf-l-page__header-brand-link--brand--MaxHeight); // Hardcode brand image max-height so it always aligns with everthing else.
```

Reply:
I would also like to bring up in the CSS meeting today if that should be `c-brand` when we include a component's name in a variable (or `l-layout` for a layout). Seems like that's a good idea/

Reply:
ya I wasn't sure what was best here. I didn't like pf-c- but adding c works for me.